woocommerce/woocommerce-ios
# Updated Wormholy from v1.6.4 to v1.6.5

This PR updates [Wormholy](https://github.com/pmusolino/Wormholy) from version 1.6.4 to version 1.6.5.

## Changes in v.1.6.5
- Fix: crash on iOS 15
- Fix authentication challenge handling to fix SSL pinning issue.
- Fix: keyboard content inset bug.
- Added an option for a default filter in the search bar.

## Testing
- Run `bundle exec pod install`.
- Try to open Wormholy from the Settings screen running the app in debug. It should work like before.

Update release notes:
